pronto_quadruped_ros/src/legodo_handler_ros.cpp

```diff
@@ -108,9 +117,8 @@ void LegodoHandlerBase::getPreviousState(const StateEstimator *est)
     // take the acceleration, rotational rate and orientation from the current
     // state of the filter
     xd_ = head_state_.velocity();
-    xdd_ = head_state_.acceleration() - head_state_.orientation().inverse()*Eigen::Vector3d::UnitZ()*9.80655;  // TODO: Standardise gravitational acceleration
+    xdd_ = head_state_.acceleration() + head_state_.orientation().inverse() * pronto::g_vec;
```

Comment:
where is `pronto::g_vec` defined and is it negative?

Reply:
It's defined here: https://github.com/ori-drs/pronto/blob/be5e2b5f14594c7397a7f13f479390d87728b62f/pronto_core/include/pronto_core/rigidbody.hpp#L19